packages/example-getting-started/src/application.ts

```diff
@@ -5,39 +5,51 @@
 
 import {ApplicationConfig} from '@loopback/core';
 import {RestApplication} from '@loopback/rest';
+import {MySequence} from './sequence';
 import {db} from './datasources/db.datasource';
 
 /* tslint:disable:no-unused-variable */
-// Do not remove!
-// Class and Repository imports required to infer types in consuming code!
 // Binding and Booter imports are required to infer types for BootMixin!
 import {BootMixin, Booter, Binding} from '@loopback/boot';
 import {
   Class,
   Repository,
-  DataSourceConstructor,
   RepositoryMixin,
   juggler,
+  DataSourceConstructor,
 } from '@loopback/repository';
 /* tslint:enable:no-unused-variable */
 
-export class TodoApplication extends BootMixin(
+export class TodoListApplication extends BootMixin(
   RepositoryMixin(RestApplication),
 ) {
   constructor(options?: ApplicationConfig) {
     super(options);
+
+    // Set up the custom sequence
+    this.sequence(MySequence);
+
     this.projectRoot = __dirname;
-    this.setupDataSources();
+    // Customize @loopback/boot Booter Conventions here
+    this.bootOptions = {
+      controllers: {
+        // Customize ControllerBooter Conventions here
+        dirs: ['controllers'],
+        extensions: ['.controller.js'],
+        nested: true,
+      },
+    };
+
+    this.setupDatasources();
   }
 
-  // Helper functions (just to keep things organized)
-  setupDataSources() {
-    // TODO(bajtos) Automate datasource and repo registration via @loopback/boot
-    // See https://github.com/strongloop/loopback-next/issues/441
+  setupDatasources() {
+    // This will allow you to test your application without needing to
+    // use the "real" datasource!
     const datasource =
       this.options && this.options.datasource
         ? new DataSourceConstructor(this.options.datasource)
         : db;
-    this.dataSource(datasource);
+    this.bind('datasource').to(datasource);
```

Comment:
Do we have a reason for not using the `dataSource` sugar?

Reply:
No reason, I was just going with the original instructions. Updated.